repo.yml

```diff
@@ -5,4 +5,5 @@ upstream:
   - repo: 'meta-resin'
     url: 'http://github.com/resin-os/meta-resin'
   - repo: 'meta-balena'
-    url: 'http://github.com/balena-os/meta-balena'
\ No newline at end of file
+    url: 'http://github.com/balena-os/meta-balena'  - repo: 'balena-yocto-scripts'
```

Comment:
I believe you missed a newline character?!

Reply:
Indeed, thanks.